airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -535,6 +537,7 @@ def execute(self, context: Context):
 
     def execute_sync(self, context: Context):
         result = None
+        self.killed = False
```

Comment:
are you referring to self._killed?

Reply:
yep, this should be `self._killed`. Let me fixe it